<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>knn_action</class>
 <widget class="QDialog" name="knn_action">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>InputData</string>
  </property>
  <widget class="QPushButton" name="Data_In_Button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>40</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color:rgb(62, 62, 62); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color:rgb(91, 91, 136)}</string>
   </property>
   <property name="text">
    <string>打开文件(EXCEL)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="filename">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>自变量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>300</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>因变量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据显示</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="datatable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>461</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="charatable">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>121</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="ivtable">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>120</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="dvtable">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>111</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="p_iv">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>210</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
    </font>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="p_dv">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>410</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
    </font>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>变量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pmButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>520</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color:rgb(62, 62, 62); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color:rgb(91, 91, 136)}</string>
   </property>
   <property name="text">
    <string>模型参数设置</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>love</string>
   </property>
   <widget class="QRadioButton" name="s_radio_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>119</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>首行保留</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="s_radio_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>119</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>首行去除</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
